### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by mmsesay**
In this PR I worked on the following:

- Add: styling to the application
- Add utility function to generate random percentages, chapters and sections
- Add style to the error message popup

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  background-color: #fff;
+}
+
+.Bookstore-CMS {
+  width: 100rem;
+  height: 89.625rem;
+  margin: 0 auto;
+  padding: 9.313rem 3.75rem 7.688rem 6.25rem;
+  background-color: var(--pale-grey);
+}
```
Comment (SarvarKh) [inline src/styles/main.css]: Your style looks almost identical to the one shown in the Zeplin design.

However, one of the project requirements was Your final design should match Zeplin's design.
Therefore, we must stick to the requirement. Currently, your Bookstore app is missing pale grey background around your main app.

- [x] Please add the background as in the Zeplin;
- [x] You have a horizontal overflow, please remove it;

Here is how it looks like as per the Zeplin (notice there is grey background around main app window).
Screen Shot 2021-11-05 at 23 47 00

File: src/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+.Text-Style-5 {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.flex {
+  display: flex;
+}
+
+.panel-bg {
```
Comment (SarvarKh) [inline src/styles/main.css:21]: Your final design should match Zeplin's design.

- [x] Please make your navbar exactly the same as in the Zeplin (including Capitalized 'CMS', font size, missing icon, etc.);

Here is your navbar:
Screen Shot 2021-11-06 at 01 13 56

Here is how it shall look:
Screen Shot 2021-11-06 at 01 14 10

File: src/styles/main.css [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+}
+
+.Lesson-Panel {
```
Comment (SarvarKh) [inline src/styles/main.css:77]: `Lesson-Panel`'s Container's background color is not properly set.
The background inside your `.Lesson-Panel` is white while outside it is darker.

- [x] please kindly check the make the outer side of your `.Lesson-Panel` the same as in the Zeplin;

File: src/components/AddBookForm.js [hunk 4]
```diff
@@ -37,27 +50,34 @@ export const AddBookForm = () => {
...
-      <form method="POST" onSubmit={handleFormSubmit}>
-        <input 
-          type="text"
-          placeholder="Author Name" 
-          value={author}
-          name="author"
-          onChange={(e) => setAuthor(e.target.value)}
-        />
-        <br/>
+    <div className="border mx-20 p-10 mt-10">
```
Comment (SarvarKh) [inline src/components/AddBookForm.js]: Your form is not the same as in the Zeplin.

- [ ] Please kindly check Zeplin design and make sure it is the same;

Your design:
Screen Shot 2021-11-06 at 01 39 20

Zeplin's:
Screen Shot 2021-11-06 at 01 39 37

**Review by SarvarKh — CHANGES_REQUESTED** [review]
Hi @mmsesay ,

You have done a great job so far! 👏 
:heavy_check_mark: Your repository looks professional with descriptive README and commits;
:heavy_check_mark: Both Github and Git Flows were used correctly.

In order to go to the next project, you shall work on a few issues blocking your project from excellence.
You have well-crafted your site and are one step behind its completeness :trophy:

Implementing those given comments might be unpleasant, but your and my objectives are to make you a Master:sunglasses: and one of the Project requirements is your styling must be EXACTLY the same as the one provided on Zeplin.
So, do your best and give it the last shot🦾🙏


## Required Changes ♻️
_Check the comments under the review._

- [x] Please, also update your README.md file as per the given standard of the [Microverse](https://github.com/microverseinc/readme-template) (including your live demo, broken MIT license, adding screenshots of your app). This makes your project more professional. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/main.css (hunk 1 above)
Comment (mmsesay) [reply src/styles/main.css]: Thanks for the guide. I have refactor the design with the background added.

File: src/styles/main.css (hunk 2 above)
Comment (mmsesay) [reply src/styles/main.css:21]: Thanks for the review. I have fixed up the navbar design to match Zeplin's

File: src/styles/main.css (hunk 3 above)
Comment (mmsesay) [reply src/styles/main.css:77]: Yeah, I get it. I have fixed the background.

File: src/components/AddBookForm.js (hunk 4 above)
Comment (mmsesay) [reply src/components/AddBookForm.js]: Thanks I have updated the design as suggested.

**Review by afanimartin — APPROVED** [review]
Hi @mmsesay, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**